tools/clang/test/SemaHLSL/attributes/wavesize_range.ll

```diff
@@ -0,0 +1,51 @@
+; RUN: %dxv %s | FileCheck %s
```

Comment:
Validation tests need to be under `tools/clang/test/HLSLFileCheck/validation` for now because there's no way to predicate certain tests on validator version here.
Also, you need to add `%dxilver 1.8 | ` at the beginning of the RUN line.